card.go

```diff
@@ -79,6 +81,29 @@ func (c *Client) Card(CardId string) (card *Card, err error) {
 	return
 }
 
+// NewCard creates with the attributes of the supplied Card struct
+// https://developers.trello.com/advanced-reference/card#post-1-cards
+func (c *Client) NewCard(card Card) (_ *Card, err error) {
+	payload := url.Values{}
+	payload.Set("name", card.Name)
+	payload.Set("desc", card.Desc)
+	payload.Set("pos", strconv.Itoa(card.Pos))
+	payload.Set("due", card.Due)
+	payload.Set("idList", card.IdList)
+	payload.Set("idMembers", strings.Join(card.IdMembers, ","))
+
+	body, err := c.Post("/cards", payload)
+	if err != nil {
+		return nil, err
+	}
+
+	if err = json.Unmarshal(body, &card); err != nil {
+		return nil, err
+	}
+	card.client = c
+	return &card, nil
+}
+
```

Comment:
Trello accepts data in form of JSON, I'm pretty sure. But this has nothing to do with this PR, I'm using `url.Values` because all other methods use it.